docker/docker-compose.yml

```diff
@@ -37,12 +37,13 @@ services:
       - 5555:5555
       - 5556:5556
       - 5557:5557
+      - 3001:3001
     restart: unless-stopped
     tty: true
     volumes:
-      - ../src/jukebox:/root/RPi-Jukebox-RFID/src/jukebox
-      - ../shared:/root/RPi-Jukebox-RFID/shared
-      - ./config/docker.pulse.mpd.conf:/root/.config/mpd/mpd.conf
+      - ../src/jukebox:/home/pi/RPi-Jukebox-RFID/src/jukebox
+      - ../shared:/home/pi/RPi-Jukebox-RFID/shared
+      - ./config/docker.pulse.mpd.conf:/home/pi/.config/mpd/mpd.conf
```

Comment:
Same here, let's be consistent. `/home/pi` was not something we used anymore. Maybe merge the most recent `future3/develop`?

Reply:
Yes, I need to update the branch to be up to date with `future3/develop`